pkg/volume/csi/csi_mounter.go

```diff
@@ -290,29 +291,12 @@ func (c *csiMountMgr) TearDownAt(dir string) error {
 		return nil
 	}
 
-	// load volume info from file
```

Comment:
Why is this code being moved to `NewUnmounter()`?

Reply:
Follow up, why isn't `saveVolumeData(...)` also being moved?

Reply:
initialization of `csiClient` is moved to `New{Moun|Unmoun}ter` to keep value injection of `csiMounterMgr` consistent and in one location, where possible.  This also helps clean the method of non-storage code.  I agree, `saveVolumeData` probably should be moved.

Reply:
Ok, `saveVolumeData` will stay. This is because it's called in the `SetUpAt()` method after the mount dir its created which is the location where the json file is saved.

Reply:
Let's open up a an issue to clean this up later. It's really odd that save is done in one location and load is done in a completely different place.

Reply:
Once you pointed it out, I could not resist to make things more uniformed. Good suggestion (included a bit of clean up too)  Done.